spec/unit/packagers/rpm_spec.rb

```diff
@@ -217,6 +218,7 @@ module Omnibus
           expect(contents).to include("/.file1")
           expect(contents).to include("%dir /dir2")
           expect(contents).to include("/file2")
+          expect(contents).to include("%dir \"/dir3 space\"")
```

Comment:
I'm probably just being daft, but the change above seems to say there shouldn't be spaces here?